src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java

```diff
@@ -125,7 +125,7 @@ private ProfileSAMLResult doRequest(Instant startInstant) throws IOException {
         Instant sessionExpiry = startInstant.plus(assumeRequest.getDurationSeconds() - 30, ChronoUnit.SECONDS);
         AssumeRoleWithSAMLResult assumeResult = roleHelper.assumeChosenAwsRole(assumeRequest);
 
-        String profileName = profileHelper.getProfileName(assumeResult, environment.oktaProfile);
+        String profileName = profileHelper.getProfileName(assumeResult);
```

Comment:
:art: the environment is already in scope of the call, so there's no need to hand it in.